.github/workflows/test.yml

```diff
@@ -24,7 +24,7 @@ jobs:
         id: test-dirs
         run: |
           cd app/tests/
-          export TESTS=$(ls -d * | jq -R -s -c 'split("\n")[:-1]')
+          export TESTS=$(ls -d * | grep -v ble | jq -R -s -c 'split("\n")[:-1]')
```

Comment:
Yeah... That's what I'd like to avoid. Not sure I want to pull that trigger *now*. I think we do it whenever we actually have one more added that gives us issues. Any concern?

Reply:
Nope. If we start adding more folders that need to be excluded later, then we can come up with something.